mne/evoked.py

```diff
@@ -1354,7 +1354,7 @@ def write_evokeds(fname, evoked):
 
 def _write_evokeds(fname, evoked, check=True):
     """Write evoked data."""
-    from .epochs import _compare_epochs_infos
+    from .io.meas_info import _ensure_infos_match
```

Comment:
this does not need to be nested anymore and can be combined with the existing `from .io.meas_info` line at the top